.github/workflows/main.yml

```diff
@@ -1997,6 +2000,12 @@ jobs:
       - name: tomcat5
         run: ant $OPTS -f enterprise/tomcat5 test
 
+      - name: enterprise/web.jsf
+        run: ant $OPTS -f enterprise/web.jsf test
```

Comment:
`testPrettyFacesModel: org.netbeans.modules.web.jsf.metamodel.SeveralXmlModelTest` failed on first attempt but worked after restart.
https://github.com/apache/netbeans/actions/runs/5493785724/attempts/2
https://github.com/apache/netbeans/actions/runs/5493785724/attempts/3

lets wrap this into the retry script to reduce the whack-a-mole factor
```yaml
       run: .github/retry.sh ant $OPTS -f enterprise/web.jsf test
```

Reply:
Done!